server/src/main/java/org/elasticsearch/action/bulk/BulkResponse.java

```diff
@@ -49,28 +50,42 @@
     private static final String ERRORS = "errors";
     private static final String TOOK = "took";
     private static final String INGEST_TOOK = "ingest_took";
+    private static final String ITEMS_OMITTED = "items_omitted";
```

Comment:
maybe rename to `no_items`? Then it is more inline with the field name `noItemsOnSuccess `?